rules.go

```diff
@@ -1762,7 +1760,12 @@ func isValidLiteralValue(ttype Input, valueAST ast.Value) (bool, []string) {
 		if valueAST, ok := valueAST.(*ast.ListValue); ok {
 			messagesReduce := []string{}
 			for idx, value := range valueAST.Values {
-				_, messages := isValidLiteralValue(itemType, value)
+				var messages []string
+				if value.GetKind() == kinds.NullValue {
+					messages = []string{"Unexpected null literal."}
```

Comment:
@mpenick Is there a reason for this disallowing null in a list? There's even a test for it so it doesn't seem to be an accident.

Section 2.9.7 seems to allow nulls in a list.
https://spec.graphql.org/June2018/#ListValue